t/01-driver.t

```diff
@@ -450,6 +450,12 @@ STORAGE: {
     }
 }
 
+UPLOAD: {
+    #Webdriver only returns the full filename if there isn't any path components in it, so test both cases
+    like( $driver->upload_file('uploadTest'),qr/uploadTest$/,'upload_file returns FULL path to the file: cwd');
+    like( $driver->upload_file('t/uploadTest'),qr/uploadTest$/,'upload_file returns FULL path to the file: subdir');
```

Comment:
These tests catch 2 particular problems:

1. The current behavior is to return 1 rather than the path at all.
2. If I pass 'somefile' as the argument, selenium returns /tmp/uuid/path/to/file/somefile.
    If I pass subdir/somefile as the argument, selenium returns /tmp/uuid/path/to/file/subdir, and omits the filename.  Rather than document this, I thought it was appropriate to make the upload_file method "do the right thing".

The test should not necesarily be specific beyond the filename; the path is in a temporary directory, which will vary across platforms; even File::Spec->tmpdir is not reliable in my experience.  Furthermore, it uses a randomly generated UUID which I cannot know beforehand in the path.

Probably the only more specificity the test could have would be checking that the session ID is a part of the path somewhere...